gpytorch/utils/cholesky.py

```diff
@@ -8,102 +8,57 @@
-
-    def _psd_safe_cholesky(A, out=None, jitter=None, max_tries=3):
-        # Maybe log
-        if settings.verbose_linalg.on():
-            settings.verbose_linalg.logger.debug(f"Running Cholesky on a matrix of size {A.shape}.")
-
-        try:
-            L = torch.linalg.cholesky(A, out=out)
-            return L
-        except RuntimeError as e:
-            isnan = torch.isnan(A)
-            if isnan.any():
-                raise NanError(
-                    f"cholesky_cpu: {isnan.sum().item()} of {A.numel()} elements of the {A.shape} tensor are NaN."
-                )
-
-            if jitter is None:
-                jitter = settings.cholesky_jitter.value(A.dtype)
-            Aprime = A.clone()
-            jitter_prev = 0
-            for i in range(max_tries):
-                jitter_new = jitter * (10 ** i)
-                Aprime.diagonal(dim1=-2, dim2=-1).add_(jitter_new - jitter_prev)
-                jitter_prev = jitter_new
-                try:
-                    L = torch.linalg.cholesky(Aprime, out=out)
-                    warnings.warn(f"A not p.d., added jitter of {jitter_new:.1e} to the diagonal", NumericalWarning)
-                    return L
-                except RuntimeError:
-                    continue
-            raise NotPSDError(
-                f"Matrix not positive definite after repeatedly adding jitter up to {jitter_new:.1e}. "
-                f"Original error on first attempt: {e}"
-            )
+    raise NotPSDError(f"Matrix not positive definite after repeatedly adding jitter up to {jitter_new:.1e}.")
 
 
 def psd_safe_cholesky(A, upper=False, out=None, jitter=None, max_tries=3):
     """Compute the Cholesky decomposition of A. If A is only p.s.d, add a small jitter to the diagonal.
     Args:
-        :attr:`A` (Tensor):
-            The tensor to compute the Cholesky decomposition of
-        :attr:`upper` (bool, optional):
-            See torch.cholesky
-        :attr:`out` (Tensor, optional):
-            See torch.cholesky
-        :attr:`jitter` (float, optional):
-            The jitter to add to the diagonal of A in case A is only p.s.d. If omitted,
-            uses settings.cholesky_jitter.value()
-        :attr:`max_tries` (int, optional):
-            Number of attempts (with successively increasing jitter) to make before raising an error.
+            :attr:`A` (Tensor):
+                    The tensor to compute the Cholesky decomposition of
+            :attr:`upper` (bool, optional):
+                    See torch.cholesky
+            :attr:`out` (Tensor, optional):
+                    See torch.cholesky
+            :attr:`jitter` (float, optional):
+                    The jitter to add to the diagonal of A in case A is only p.s.d. If omitted,
+                    uses settings.cholesky_jitter.value()
+            :attr:`max_tries` (int, optional):
+                    Number of attempts (with successively increasing jitter) to make before raising an error.
```

Comment:
what happened here?

Reply:
Just `black`.

Reply:
This doesn't seem like a change that black should suggest? Does black complain when this is undone?

Reply:
OK, I see what's going on now, and this probably explains the weird problems I had with the CI in the other PR.

Locally I have filters on smudge/clean to convert spaces to tabs. `black` saw these tabs and replaced with 8 spaces. Otherwise, `black` leaves leading spaces in docstrings unchanged.

Fixed.